contents/blog/what-is-a-growth-engineer.md

```diff
@@ -0,0 +1,134 @@
+---
+title: What is a growth engineer (and how they help your startup)
+date: 2023-04-20
+author:
+    - ian-vanagas
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Engineering
+tags:
+ - Explainers
+---
+
+Every startup loves to see its growth metrics like signups, subscriptions, and revenue go up, yet nearly all engineers work on things indirectly related to doing so. Growth engineers are different. They focus entirely on projects dedicated to driving as much growth as possible, no matter where they build those projects.
+
+The post goes over growth engineers, with a specific focus on our technical version role at PostHog. We cover the type of people who succeed as growth engineers, their skills, what they work on, and why you should hire them.
+
+## What are the characteristics of a growth engineer?
+
+To start, it is useful to understand who growth engineers are as people. When comparing them to regular engineers, here are the characteristics that stand out:
```

Comment:
something reads strange to me about the phrase "regular engineers". Makes them sound somewhat inferior to growth engineers 😂

Reply:
Changed 😅